components/raftstore/src/store/worker/raftlog_fetch.rs

```diff
@@ -19,7 +22,15 @@ pub enum Task {
         tried_cnt: usize,
         term: u64,
     },
-    // More to support, suck as fetch entries ayschronously when apply and schedule merge
+
+    // GenTabletSnapshot is used to generate tablet snapshot.
+    GenTabletSnapshot {
```

Comment:
There will be no states in tablet, so the metadata needs to be either stored in `Storage` or in the task.

Reply:
Do you mean `apply state` or else? I will add it to the next PR.

Reply:
Apply state and region state.